src/Traits/VoyagerUser.php

```diff
@@ -46,15 +78,9 @@ public function setRole($name)
 
     public function hasPermission($name)
     {
-        if (!$this->relationLoaded('role')) {
-            $this->load('role');
-        }
+        $this->loadPermissionsCache();
```

Comment:
While I completely understand the reason behind this, is it absolutely necessary for this PR? Can it be implemented on it's own then modified for this PR? "Caching Permissions" seems 100% unrelated to "Multiple Roles"

Reply:
You're right, I also dislike that approach. Maybe we can skip this cache variable.
Will check that.

Thanks @fletch3555 for your feedback! :)